dropwizard-jdbi/src/test/java/io/dropwizard/jdbi/timestamps/DateTimeSqlTimestampTest.java

```diff
@@ -101,11 +101,10 @@ public void testReadTimestamp() {
 
     public interface FlightDao {
 
-        @SqlUpdate("INSERT INTO flights(flight_id, departure_airport, arrival_airport,departure_time, arrival_time) " +
-                "VALUES (:flight_id, :departure_airport, :arrival_airport, :departure_time, :arrival_time)")
-        int insert(@Bind("flight_id") String flightId, @Bind("departure_airport") String departureAirport,
-                   @Bind("arrival_airport") String arrivalAirport,
-                   @Bind("departure_time") DateTime departureTime, @Bind("arrival_time") DateTime arrivalTime);
+        @SqlUpdate("INSERT INTO flights(flight_id, departure_airport, arrival_airport, departure_time, arrival_time) " +
+                "VALUES (:0, :1, :2, :3, :4)")
```

Comment:
Is this change really necessary? I think named parameters are way less error-prone than mere numbered parameters in this case.